readme.md

```diff
@@ -62,31 +120,9 @@ f27548f03a4179b7a4dc3c8a62fcfc5f84be15ff@35.166.247.98:28004
 dd35505768be507af3c76f5a4ecdb272537e398f@35.166.247.98:28006
 ```
 
-# Verfiy
```

Comment:
why you dropping this bit?

Reply:
I guess the thinking was that it won't exactly match what we had there, but perhaps I can re-word it. I'll add something else.